packages/frontend2/src/server/features/bridges/types.ts

```diff
@@ -0,0 +1,36 @@
+import { type ScalingProjectRiskViewEntry } from '@l2beat/config'
+import { type TokenBreakdownProps } from '~/app/_components/breakdown/token-breakdown'
+
+export type CommonBridgeEntry = {
```

Comment:
Don't we want to follow the same approach as I used in new pages, and just define entries as `ReturnType` of the functions that create them?

Reply:
I'll take care of that as a part of the bridges summary page - I'm not entirely sure about removing these entry-types since those serve as a kind of contract between actual frontend components and data we have to provide.